integration-tests/api-gateway/api-test-context/load-integration-context.ts

```diff
@@ -0,0 +1,8 @@
+import { promises as fs } from "fs";
+import path from "path";
+
+export const loadIntegrationContext = async () => {
+  const filePath = path.join(__dirname, "integration-context.json");
```

Comment:
How does this work? I can't find a file called `integration-context.json` anywhere

Reply:
I gets created at the beginning and deleted after test